msg/service.go

```diff
@@ -12,6 +13,10 @@ import (
 	"github.com/miekg/dns"
 )
 
+var PathPrefix string
```

Comment:
Wouldn't it be easier to let user just override this var?

```
var PathPrefix string = "skydns"
```

With a proper comment?